pkg/cmd/cluster/bind/bind.go

```diff
@@ -69,6 +70,7 @@ func NewBindCommand(f *factory.Factory) *cobra.Command {
 	cmd.Flags().BoolVarP(&opts.forceCreationWithoutAsk, "yes", "y", false, localizer.MustLocalizeFromID("cluster.common.flag.yes.description"))
 	cmd.Flags().StringVarP(&opts.namespace, "namespace", "n", "", localizer.MustLocalizeFromID("cluster.common.flag.namespace.description"))
 	cmd.Flags().BoolVarP(&opts.ignoreContext, "ignore-context", "", false, localizer.MustLocalizeFromID("cluster.common.flag.ignoreContext.description"))
+	cmd.Flags().BoolVarP(&opts.useOperator, "force-operator", "", false, localizer.MustLocalizeFromID("cluster.bind.flag.onlyOperator.description"))
```

Comment:
@pmuir We assume to use Operator by default when installed. If this flag is added and CLI fails to detect operator it will fail. Otherwise if operator is not detected SDK is used. 

Would you prefer to have more complex flags like:
```
--useOperator=true
--useSDK=true
```

Additionally we should provide name override for the service binding (as it is currently generated which might look bad)

Reply:
I like `--force-operator` - it's clear what's it's doing :-D What about `--force-sdk` as well?

Reply:
Yes. Adding SDK here will be easy. I was hoping to have single flag but that is not possible due to this fallback logic